src/Helius.ts

```diff
@@ -316,6 +318,14 @@ export class Helius {
     }
   }
 
+  /** Get the API endpoint for the specified path */
+  getEndpoint(path: string): string {
```

Comment:
Maybe this should be called `getApiEndpoint` to make it more explicit what it's for.

Reply:
Yeah, if you could rename it to `getApiEndpoint` so its more explicit that would be greatly appreciated!